src/main/kotlin/com/safehill/kclient/network/remote/RemoteServer.kt

```diff
@@ -386,10 +400,12 @@ class RemoteServer(
             this.requestor.authToken ?: throw SafehillError.ClientError.Unauthorized
 
         val assetCreatedAt = asset.creationDate ?: run { Instant.MIN }
+        val dateTime = OffsetDateTime.ofInstant(assetCreatedAt, ZoneOffset.UTC)
+        val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSX")
         val requestBody = com.safehill.kclient.models.dtos.AssetInputDTO(
             asset.globalIdentifier,
             asset.localIdentifier,
-            assetCreatedAt,
+            dateTime.format(formatter),
```

Comment:
Are we using different format (not  ISO-8601) for uploading?

Reply:
We should use this one for everything: DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSX") It is ISO-8601. It just adds the milliseconds

Reply:
Cannot we use the builtin DateTimeFormatter [ISO_DATE_TIME](https://docs.oracle.com/en%2Fjava%2Fjavase%2F22%2Fdocs%2Fapi%2F%2F/java.base/java/time/format/DateTimeFormatter.html#ISO_DATE_TIME)?